Gentoo symlinks various blas implementations to blas.pc. Use that.
Bug: https://github.com/gentoo-haskell/gentoo-haskell/issues/446
diff --git a/hmatrix.cabal b/hmatrix.cabal
index 3895dc1..a2789e0 100644
--- a/hmatrix.cabal
+++ b/hmatrix.cabal
@@ -34,7 +34,2 @@ extra-source-files: src/C/lapack-aux.h
 
-flag openblas
-    description:    Link with OpenBLAS (https://github.com/xianyi/OpenBLAS) optimized libraries.
-    default:        False
-    manual: True 
-
 library
@@ -107,24 +102,3 @@ library
 
-    if flag(openblas)
-        extra-lib-dirs:     /usr/lib/openblas/lib    
-        extra-libraries:    openblas
-    else 
-        extra-libraries:    blas lapack
-
-    if os(OSX)
-        extra-lib-dirs: /opt/local/lib/
-        include-dirs: /opt/local/include/
-        extra-lib-dirs: /usr/local/lib/
-        include-dirs: /usr/local/include/
-        if arch(i386)
-            cc-options: -arch i386
-        frameworks: Accelerate
-
-    if os(freebsd)
-       extra-lib-dirs: /usr/local/lib
-       include-dirs: /usr/local/include
-       extra-libraries: blas lapack gfortran
-
-    if os(windows)
-        extra-libraries: blas lapack
+    pkgconfig-depends: blas, lapack
 
